frontend/apps/reader/modules/readergesture.lua

```diff
@@ -259,7 +259,7 @@ function ReaderGesture:init()
             end
         end
         if reset then
-            logger.info("UI language direction changed: resetting some gestures to direction default")
+            logger.info("UI language direction changed: Resetting some gestures to default direction.")
```

Comment:
While I don't typically police comments too much, our house style doesn't capitalize after colons.
```suggestion
            logger.info("UI language direction changed: resetting some gestures to default direction.")
```
```suggestion
            logger.info("UI language direction changed. Resetting some gestures to default direction.")
```

Reply:
No, it should stay `resetting some gestures to direction default`.
We're not resetting gestures to any "direction" (so, not to "default direction).
We're resetting gestures to the gestures that are the default for that new direction.

Reply:
Pretty sure I saw both. Using the one that is only allowed in English baits errors in translation.

Reply:
Could you please point out the problematic strings? This is just random code.

Reply:
Indeed, that's just some string that is logged in crash.log - it shouldn't reach weblate in any way - or does it ?!

Reply:
@Frenzie Didn't find any, but most are unexplained placeholders, so could be I remembered it as being one of those. https://github.com/koreader/koreader/blob/a7def7e8338659e51f4bc956487787391b76a41a/frontend/apps/reader/modules/readerwikipedia.lua#L484 is off, and I saw about 10 other clear errors looking for it.
Do you want me to find evidence of errors in translation resulting from : small-versal?

Reply:
Yes please! :-)